src/container/AudioPlayer.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Animated } from 'react-native';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as Actions from '../actions/index';
+import styled from 'styled-components/native';
+
+import AudioP from '../components/AudioPlayer';
+import perpareReciterSurahsList from '../utils/perpareReciterSurahsList';
+import FontAwesome from 'react-native-vector-icons/FontAwesome';
+
+const View = styled.View`
+  height: ${props => props.height}px;
+  width: 100%;
+  background: #000;
+  borderTopColor: #2ca4ab;
+  borderTopWidth: 5px;
+`;
+const AView = Animated.createAnimatedComponent(View);
+
+const ViewCloseIcon = styled.TouchableOpacity`
+  background: #2ca4ab;
+  width: 30px;
+  height: 30px;
+  align-self: flex-end;
+  align-items: center;
+  justify-content: center;
+  border-radius: 5px;
+  position: relative;
+  top: -3px;
+`;
+
+const Chevron = styled(FontAwesome)`
+ transform: rotate(${props => props.deg});
+`;
+
+class AudioPlayer extends React.Component {
+  render() {
+    const { chapters, player, actions } = this.props;
+    const selection = chapters.selection;
+    if (selection && selection.chapter) {
+      return (
+        <AView height={player.minimise ? '50' : '200'}>
+          <ViewCloseIcon onPress={() => actions.minimisePlayer()}>
```

Comment:
and these default values too might be moved to a class that wraps such constants